.github/workflows/build.yml

```diff
@@ -0,0 +1,84 @@
+name: Build
+
+on:
+  workflow_dispatch:
+  pull_request:
+    types: [opened, synchronize]
+
+jobs:
+  build:
+    name: Build library
+    runs-on: macos-latest
```

Comment:
why macos-latest? 
1. Do we have that label on agents?
2. Cant it be ubuntu?

Reply:
where i can find defined labels on agents ?
cannot find the criteria which tells me what system i should choose

Reply:
![image](https://user-images.githubusercontent.com/100935765/169802610-d3c72d50-0b3c-4388-8319-7d169fe7f241.png)
maybe this make a difference when it comes to ```macos-latest```

Reply:
ok, let it be like that if there is another pool for public repo